.github/workflows/dlp-pipelines.yml

```diff
@@ -10,14 +10,15 @@ env:
   PROJECT_ID: "dlp-dataflow-deid-ci-392604"
   DATASET_ID: "demo_dataset"
   GCS_BUCKET: "dlp-dataflow-deid-ci-392604-demo-data"
+  GCS_NOTIFICATION_TOPIC: "projects/dlp-dataflow-deid-ci-392604/topics/dlp-dataflow-deid-ci-gcs-notification-topic"
```

Comment:
Suggestion: Is it possible to utilize the PUBSUB_TOPIC_NAME instead?

Reply:
I feel the reid topic and input topic should be kept separate. While it is possible to utilize the Reid topic but it will lead to ingress and egress through same topic leading to confusion as well as performance degradation.